test/generic/PermGroupAPI-test.jl

```diff
@@ -0,0 +1,10 @@
+import GroupsCore
+
+@testset "GroupsCore API PermGroup" begin
+
+    include(joinpath(pathof(GroupsCore), "..", "..", "test", "conformance_test.jl"))
```

Comment:
To be pedantic, I think that should be:
```suggestion
    include(joinpath(dirname(pathof(GroupsCore)), "..", "test", "conformance_test.jl"))
```

Reply:
then maybe
```suggestion
    include(joinpath(dirname(dirname(pathof(GroupsCore))), "test", "conformance_test.jl"))
```